2014-08-30 17:39:37,243 ERROR [org.pentaho.platform.plugin.action.sql.SQLLookupRule] Error Start: Pentaho Pentaho Platform Core 5.0.1-stable.-1
2014-08-30 17:39:37,244 ERROR [org.pentaho.platform.plugin.action.sql.SQLLookupRule] c0f9affa-3085-11e4-8fcb-881fa1026572:COMPONENT:context-2073823600-1409431175455:global-department-list.xactionConnectFactory.ERROR_0001 - Database connection could not be established to: SampleData
org.pentaho.platform.api.data.DBDatasourceServiceException: java.sql.SQLException: socket creation error
	at org.pentaho.platform.engine.services.connection.datasource.dbcp.PooledDatasourceHelper.setupPooledDataSource(PooledDatasourceHelper.java:164)
	at org.pentaho.platform.engine.services.connection.datasource.dbcp.PooledOrJndiDatasourceService.retrieve(PooledOrJndiDatasourceService.java:47)
	at org.pentaho.platform.engine.services.connection.datasource.dbcp.PooledOrJndiDatasourceService.getDataSource(PooledOrJndiDatasourceService.java:110)
	at org.pentaho.platform.plugin.services.connections.sql.SQLConnection.initWithJNDI(SQLConnection.java:192)
	at org.pentaho.platform.plugin.services.connections.sql.SQLConnection.setProperties(SQLConnection.java:106)
	at org.pentaho.platform.engine.services.connection.PentahoConnectionFactory.getConnection(PentahoConnectionFactory.java:129)
	at org.pentaho.platform.engine.services.connection.PentahoConnectionFactory.getConnection(PentahoConnectionFactory.java:73)
	at org.pentaho.platform.plugin.action.sql.SQLBaseComponent.getConnection(SQLBaseComponent.java:893)
	at org.pentaho.platform.plugin.action.sql.SQLBaseComponent.getConnection(SQLBaseComponent.java:859)
	at org.pentaho.platform.plugin.action.sql.SQLBaseComponent.getDatasourceConnection(SQLBaseComponent.java:809)
	at org.pentaho.platform.plugin.action.sql.SQLBaseComponent.executeAction(SQLBaseComponent.java:211)
	at org.pentaho.platform.engine.services.solution.ComponentBase.execute(ComponentBase.java:465)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeComponent(RuntimeContext.java:1313)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeAction(RuntimeContext.java:1279)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.performActions(RuntimeContext.java:1176)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeLoop(RuntimeContext.java:1122)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeSequence(RuntimeContext.java:1004)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeSequence(RuntimeContext.java:910)
	at org.pentaho.platform.engine.services.solution.SolutionEngine.executeInternal(SolutionEngine.java:386)
	at org.pentaho.platform.engine.services.solution.SolutionEngine.execute(SolutionEngine.java:305)
	at org.pentaho.platform.engine.services.solution.SolutionEngine.execute(SolutionEngine.java:184)
	at org.pentaho.platform.engine.core.system.PentahoSystem.globalStartup(PentahoSystem.java:868)
	at org.pentaho.platform.engine.core.system.PentahoSystem$2.call(PentahoSystem.java:825)
	at org.pentaho.platform.engine.core.system.PentahoSystem$2.call(PentahoSystem.java:822)
	at org.pentaho.platform.engine.security.SecurityHelper.runAsSystem(SecurityHelper.java:333)
	at org.pentaho.platform.engine.core.system.PentahoSystem.globalStartup(PentahoSystem.java:822)
	at org.pentaho.platform.engine.core.system.PentahoSystem.init(PentahoSystem.java:282)
	at org.pentaho.platform.engine.core.system.PentahoSystem.init(PentahoSystem.java:182)
	at org.pentaho.platform.web.http.context.SolutionContextListener.contextInitialized(SolutionContextListener.java:136)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4206)
	at org.apache.catalina.core.StandardContext.start(StandardContext.java:4705)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:799)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:779)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:601)
	at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:675)
	at org.apache.catalina.startup.HostConfig.deployDescriptors(HostConfig.java:601)
	at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:502)
	at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1317)
	at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:324)
	at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:142)
	at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1065)
	at org.apache.catalina.core.StandardHost.start(StandardHost.java:840)
	at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1057)
	at org.apache.catalina.core.StandardEngine.start(StandardEngine.java:463)
	at org.apache.catalina.core.StandardService.start(StandardService.java:525)
	at org.apache.catalina.core.StandardServer.start(StandardServer.java:754)
	at org.apache.catalina.startup.Catalina.start(Catalina.java:595)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:289)
	at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:414)
Caused by: java.sql.SQLException: socket creation error
	at org.hsqldb.jdbc.Util.sqlException(Unknown Source)
	at org.hsqldb.jdbc.jdbcConnection.<init>(Unknown Source)
	at org.hsqldb.jdbcDriver.getConnection(Unknown Source)
	at org.hsqldb.jdbcDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.addObject(GenericObjectPool.java:1679)
	at org.pentaho.platform.engine.services.connection.datasource.dbcp.PooledDatasourceHelper.setupPooledDataSource(PooledDatasourceHelper.java:150)
	... 52 more
2014-08-30 17:39:37,248 ERROR [org.pentaho.platform.plugin.action.sql.SQLLookupRule] Error end:
2014-08-30 17:39:37,249 ERROR [org.pentaho.platform.plugin.action.sql.SQLLookupRule] c0f9affa-3085-11e4-8fcb-881fa1026572:COMPONENT:context-2073823600-1409431175455:global-department-list.xactionSQLBaseComponent.ERROR_0007 - Connection to data source is not valid
2014-08-30 17:39:37,250 ERROR [org.pentaho.platform.engine.services.solution.SolutionEngine] c0f9affa-3085-11e4-8fcb-881fa1026572:SOLUTION-ENGINE:/public/bi-developers/Secure/global-department-list.xaction: Action Sequence execution failed, see details below
| Error Time: Saturday, August 30, 2014 5:39:37 PM BRT
| Session ID: admin
| Instance Id: c0f9affa-3085-11e4-8fcb-881fa1026572
| Action Sequence: global-department-list.xaction
| Execution Stack:
EXECUTING ACTION: Query For Departments (SQLLookupRule)
| Action Class: SQLLookupRule
| Action Desc: Query For Departments
| Loop Index: 0
Stack Trace:org.pentaho.platform.api.engine.ActionExecutionException: RuntimeContext.ERROR_0017 - Action failed to execute
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeComponent(RuntimeContext.java:1341)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeAction(RuntimeContext.java:1279)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.performActions(RuntimeContext.java:1176)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeLoop(RuntimeContext.java:1122)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeSequence(RuntimeContext.java:1004)
	at org.pentaho.platform.engine.services.runtime.RuntimeContext.executeSequence(RuntimeContext.java:910)
	at org.pentaho.platform.engine.services.solution.SolutionEngine.executeInternal(SolutionEngine.java:386)
	at org.pentaho.platform.engine.services.solution.SolutionEngine.execute(SolutionEngine.java:305)
	at org.pentaho.platform.engine.services.solution.SolutionEngine.execute(SolutionEngine.java:184)
	at org.pentaho.platform.engine.core.system.PentahoSystem.globalStartup(PentahoSystem.java:868)
	at org.pentaho.platform.engine.core.system.PentahoSystem$2.call(PentahoSystem.java:825)
	at org.pentaho.platform.engine.core.system.PentahoSystem$2.call(PentahoSystem.java:822)
	at org.pentaho.platform.engine.security.SecurityHelper.runAsSystem(SecurityHelper.java:333)
	at org.pentaho.platform.engine.core.system.PentahoSystem.globalStartup(PentahoSystem.java:822)
	at org.pentaho.platform.engine.core.system.PentahoSystem.init(PentahoSystem.java:282)
	at org.pentaho.platform.engine.core.system.PentahoSystem.init(PentahoSystem.java:182)
	at org.pentaho.platform.web.http.context.SolutionContextListener.contextInitialized(SolutionContextListener.java:136)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4206)
	at org.apache.catalina.core.StandardContext.start(StandardContext.java:4705)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:799)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:779)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:601)
	at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:675)
	at org.apache.catalina.startup.HostConfig.deployDescriptors(HostConfig.java:601)
	at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:502)
	at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1317)
	at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:324)
	at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:142)
	at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1065)
	at org.apache.catalina.core.StandardHost.start(StandardHost.java:840)
	at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1057)
	at org.apache.catalina.core.StandardEngine.start(StandardEngine.java:463)
	at org.apache.catalina.core.StandardService.start(StandardService.java:525)
	at org.apache.catalina.core.StandardServer.start(StandardServer.java:754)
	at org.apache.catalina.startup.Catalina.start(Catalina.java:595)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:289)
	at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:414)

2014-08-30 17:40:25,968 ERROR [org.pentaho.platform.web.servlet.GenericServlet] GenericServlet.ERROR_0004 - Resource /saiku-ui/saikuplugin.properties.properties not found in plugin saiku
2014-08-30 17:40:25,983 ERROR [org.pentaho.platform.web.servlet.GenericServlet] GenericServlet.ERROR_0004 - Resource /saiku-ui/saikuplugin.properties_en.properties not found in plugin saiku
2014-08-30 17:40:25,998 ERROR [org.pentaho.platform.web.servlet.GenericServlet] GenericServlet.ERROR_0004 - Resource /saiku-ui/saikuplugin.properties_en-US.properties not found in plugin saiku
